examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -41,7 +42,7 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       }
     };
   })
-  .service('cloudberry', function($timeout, cloudberryConfig, MapResultCache) {
+  .service('cloudberry', function($timeout, $rootScope, cloudberryConfig, MapResultCache) {
```

Comment:
Why does $rootScope need to be involved? I saw the logic (line 265-270). But, no map types uses rootScope to get a result? Can we follow that logic?